app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -328,6 +334,12 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
                             "config_screen/show" -> startActivity(
                                 SettingsActivity.newInstance(this@WebViewActivity)
                             )
+                            "tag/write" -> startActivity(
```

Comment:
Just a note that I just updated the proposal. We're going to respond with an empty object here to indicate when it's done.

Reply:
Done.